packages/ra-core/src/core/CoreAdminRoutes.spec.tsx

```diff
@@ -340,7 +309,7 @@ describe('<CoreAdminRoutes>', () => {
                 </CoreAdminContext>
             );
             expect(screen.queryByText('PostList')).toBeNull();
-            expect(screen.queryByText('Loading')).toBeNull();
+            expect(screen.queryByText('Loading')).not.toBeNull();
```

Comment:
As there is no delay, the test loader (which is NOT our `<Loading>` component) is shown right away